backend/src/main/kotlin/app/ehrenamtskarte/backend/auth/service/Authorizer.kt

```diff
@@ -39,4 +39,27 @@ object Authorizer {
         return (user?.role == Role.REGION_ADMIN.db_value && user.regionId == region.id) ||
             mayViewUsersInProject(user, region.projectId.value)
     }
+
+    fun mayCreateAdministrator(
```

Comment:
I'm still really confused by the word Administrator vs User :D

Reply:
Yeah, that's true. Especially with the roles now 😢

Reply:
I've renamed it to `mayCreateUser` to stay consistent with the functions above.